spring-cloud-starter-tencent-polaris-router/src/main/java/com/tencent/cloud/polaris/router/PolarisLoadBalancerCompositeRule.java

```diff
@@ -198,4 +206,9 @@ public AbstractLoadBalancerRule getRule() {
 			return new ZoneAvoidanceRule();
 		}
 	}
+
+	@VisibleForTesting
```

Comment:
It is already a public method and if `@VisibleForTesting` is necessary？

Reply:
It is not recommended using test function annotations in non-test code .

Reply:
The annotation ```@VisibleForTesting``` on the method ```PolarisLoadBalancerCompositeRule#getDelegateRule``` has been deleted.